guidelines/terms/22/inactive.html

```diff
@@ -0,0 +1,8 @@
+<dt><dfn>inactive</dfn> (user interface components)</dt>
+<dd class="new">
+	<p class="change">New</p>	
+   <p>A state of a <a>User Interface Component</a> when it is not available for user interaction, such as a disabled control in HTML. An inactive user interface component is present but not currently operable using standard input mechanisms such as keyboard, pointer, or assistive technology.</p>
+<p class="note">An example inactive user interface component is a submit button at the bottom of a form which is present but cannot be activated until all the required fields in the form are completed.</p>   				
```

Comment:
What about controls that take focus but cannot be activated such as a date in a grid which is not available or  the other control types specified at https://w3c.github.io/aria-practices/#kbd_disabled_controls.

Reply:
i think that's covered by the proposed definition there, since it says nothing about whether or not the inactive component can receive focus or not...just that it's not available for user interaction (unless you mean that even the act of focusing on it is an interaction?) / operable.

Reply:
yes - I think we could have endless debates on that. Don't we all love endless debates.

Reply:
That wording from the APG really complicates things. I'm inclined to say that if something is navigable but not operable, it should not be considered disabled (and should not be given a pass on text contrast considerations).  We spent a lot of time at IBM designing read-only versions of components; it was difficult, but it is possible. There should be a way of achieving minimum contrast for unavailable options in a listbox (to use one of the APG examples), if the desire is to make them discoverable by all. Letting them be 'discoverable' programmatically but not visually seems inequitable.